src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+
+A single token can be associated with multiple roles, but only if the color value is used consistently across those roles. This allows for uniform color application across themes while maintaining full styling flexibility.
+
+With this system, all Carbon users can create their own themes by assigning new values to the established color tokens. IBM products should use one of the four IBM default themes.
+
+## Themes
+
+Themes serve as an organizational framework for color in Carbon, with each theme based on a specific primary background color. There are two default "light" themes and two default "dark" themes.  The light themes use White and Gray 10 backgrounds, and the dark themes use Gray 100 and Gray 90 backgrounds. Default color tokens are provided for each component based on the primary background color.
+
+| Theme | Primary Background       | Token                | Hex value    |    |
+| --- | ----------- | ------------------- | -------------------- | ------------------- |
+| White   | Global Background Light  | `$ui-background`        | <color-block showhex="true" size="xs">#ffffff</color-block>  |
+| Gray 10   | Global Background Light | `$ui-background`  | <color-block showhex="true" size="xs">#f3f3f3</color-block>  |
+| Gray 90  | Global Background Dark   | `$ui-background` | <color-block showhex="true" size="xs">#282828</color-block>    |
+| Gray 100   | Global Background Dark  | `$ui-background`        | <color-block showhex="true" size="xs">#171717</color-block>     |
+
+### Light theme
+
+Gray 10 components are generally paired with White backgrounds and White components with Gray 10 backgrounds. Some components, such as buttons and toggles, are common across both backgrounds. 
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 10 dropdown on White background.">
+        <img src="images/Light_theme_01.png" alt="Gray 10 dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="White dropdown on a Gray 10 background.">
+        <img src="images/Light_theme_revised.png" alt="White dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="White overflow menu on Gray 10 card.">
+        <img src="images/Light_theme_03_new.png" alt="White overflow menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Avoid use of midtones.">
+        <img src="images/Light_theme_04.png" alt="Avoid use of midtones.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Light theme sample](images/Light_theme_sample_revised.gif)
+
+
+_Components with common colors across both light UI backgrounds_
+
+
+### Dark theme
+
+Gray 90 components are generally paired with Gray 100 backgrounds and Gray 80 components with Gray 90 backgrounds. Some components, such as buttons and toggles, are common across all three dark UI backgrounds.
```

Comment:
Same edits as with the Light theme section:
"Gray 90 components are paired with Gray 100 backgrounds to make the Gray 100 Theme and Gray 80 components with Gray 90 backgrounds to make the Gray 90 Theme. Some components, such as buttons and toggles, share the same colors across both light themes."